Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/SubmissionResultPositiveOtherWarningFragment.kt

```diff
@@ -139,10 +149,28 @@ class SubmissionResultPositiveOtherWarningFragment : Fragment(),
             initiateWarningOthers()
         }
         binding.submissionPositiveOtherWarningHeader.headerButtonBack.buttonIcon.setOnClickListener {
-            navigateToSubmissionResultFragment()
+            handleSubmissionCancellation()
         }
     }
 
+    /**
+    * Opens a Dialog that warns user
+    * when they're about to cancel the submission flow
+    * @see DialogHelper
+    * @see navigateToSubmissionResultFragment
+    */
+    fun handleSubmissionCancellation() {
+        DialogHelper.showDialog(DialogHelper.DialogInstance(
+            requireActivity(),
+            R.string.submission_error_dialog_confirm_cancellation_title,
+            R.string.submission_error_dialog_confirm_cancellation_body,
+            R.string.submission_error_dialog_confirm_cancellation_button_positive,
+            R.string.submission_error_dialog_confirm_cancellation_button_negative,
+            false,
```

Comment:
Why is `cancelable=false` here? It means that the user can't exit the dialog with the backbutton, which differs from other dialogs on the same screen.

Reply:
I'll make that change.